libraries/chain_kv/include/b1/session/shared_bytes.hpp

```diff
@@ -139,6 +141,39 @@ inline shared_bytes shared_bytes::next() const {
    return eosio::session::shared_bytes(buffer.data(), buffer.size());
 }
 
+inline shared_bytes shared_bytes::next_sub_key() const {
+   auto buffer = std::vector<unsigned char>{ std::begin(*this), std::end(*this) };
+   buffer.push_back('\0');
+
+   return eosio::session::shared_bytes(buffer.data(), buffer.size());
+}
+
+inline shared_bytes shared_bytes::previous() const {
```

Comment:
He and I were working off the same branch.  My code needs it, but since I changed it since then, it would be fine to remove and then let me deal with the merge problems.